src/Websites/Websites/Websites.csproj

```diff
@@ -11,6 +11,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.KeyVault" Version="3.1.0" />
```

Comment:
As we talked, please stop referencing SDK from other RPs, as it will cause assembly confilict.
Instead, use the client from this namespace `Microsoft.Azure.Commands.Common.KeyVault.Version2016_10_1`